client/console/src/main/java/org/apache/syncope/client/console/commons/PropertyList.java

```diff
@@ -174,13 +174,11 @@ public static String getEnumValuesAsString(final List<String> enumerationValues)
         final StringBuilder builder = new StringBuilder();
```

Comment:
Why not replacing the whole method with

```
return enumerationValues.stream().
                map(v -> StringUtils.isBlank(v) ? StringUtils.EMPTY : v.trim()).
                collect(Collectors.joining(SyncopeConstants.ENUM_VALUES_SEPARATOR));
```

?

Reply:
Sure, LGTM! It should work seamlessly.